### PR TITLE
Ganache cli

### LINKED ISSUE #2: Usage of ganache-cli
Currently the provided code describes and uses a ganache instance.
Manual steps are needed to copy the proper keys to be used for the ethereum accounts into local available configuration files.
With the usage of ganache-cli, this setup can be done in an automatic way for testing purpose.

### PR DESCRIPTION
**Pull request by danielksan81**
Includes new scripts for automated setup of ganache-cli.
resolves #2

### CODE REVIEW

File: EoT-Agents-Manifacturing-Marketplace/README.md [hunk 1]
```diff
@@ -2,23 +2,26 @@
...
-- deploy the contract
-  - run the ganache client (ethereum client) either using the GUI or the console
-        ```console
-        foo@bar:~$ ganache-cli -a 2```
-  - Deploy ServiceDirectory.sol contract under folder service-directory/contracts
-        ```console
-        foo@bar:~$ truffle migrate```
+There are currently 4 agents included, two *selling_agents* and two *purchasing_agents*. These agents are already preconfigured and can be run out of the box after the testnet and smart contracts are setup as desribed in prerequisities.
+
+To run these agents the following steps has to be performed first in the *selling_agents* and the in the *purchasing agents*:
```
Comment (l-wegner) [inline EoT-Agents-Manifacturing-Marketplace/README.md:12]: ***then** in the purchasing

File: service-directory/Readme.md [hunk 2]
```diff
@@ -6,15 +6,45 @@ It is preconfigured to use a local running [Ganache](https://www.trufflesuite.co
...
+
+This configuration is also reflected in the [truffle-config.js](truffle-config.js) file in the *development* network.
+
+### Deploying contracts to ganache
+
+Migrating/deploying to Ganache can be done using `truffle migrate`.
+
+### Updating contract address in agents
+
+As the agents needs to know the current address of the newly deployed [*ServiceDirectory*](contracts/ServiceDirectory.sol), this address needs to be updated in the corresponding *skill.yaml* files.
```
Comment (l-wegner) [inline service-directory/Readme.md:30]: Should we detail, where to find that contract address?

Comment (l-wegner) [reply service-directory/Readme.md:30]: Just found the update-contract-address.sh. This should definitely be mentioned.

**Review by l-wegner — CHANGES_REQUESTED** [review]

File: service-directory/Readme.md (hunk 2 above)
Comment (danielksan81) [reply service-directory/Readme.md:30]: I see...
The idea was to describe how to update them in a manual way and then point to the script, which is described at line 40.
Maybe I have to reorder it to mention first the script and then how to do it in a manual way for custom configuration?

Comment (l-wegner) [reply service-directory/Readme.md:30]: Ah i see. I think in a rendered markdown that's clear. I got distracted by the code block and lost context, when reading line 40.

Comment (danielksan81) [reply service-directory/Readme.md:30]: I will reorganize it a little bit

**Review by l-wegner — APPROVED** [review]

**Pull request merged**